src/components/Page/AdjacentPages.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import Link from '../Link/Link';
+import './AdjacentPages.scss';
+import PropTypes from 'prop-types';
+AdjacentPages.propTypes = {
+  previous: PropTypes.shape({
+    url: PropTypes.string,
+    title: PropTypes.string,
+  }),
+  next: PropTypes.shape({
+    url: PropTypes.string,
+    title: PropTypes.string,
+  }),
+};
+
+export default function AdjacentPages({ previous, next }) {
+  return (
+    <div className="adjacent-links">
```

Comment:
Please add an if here to return `null` if none previous or next is defined given this container will add empty space to the page if rendered empty:
```scss
.adjacent-links {
  display: flex;
  padding: 30px 10px;
  justify-content: space-between;
```